composer.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "soy-php/doctrine-migrations-task",
+  "description": "Doctrine Migrations task for Soy",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Rick Kuipers",
+      "email": "io@rskuipers.com"
+    }
+  ],
+  "require": {
+    "soy-php/soy": "0.1.*"
+  },
+  "autoload": {
+    "psr-4": {
+      "Soy\\DoctrineMigrations\\": "src/Soy/DoctrineMigrations/"
+    }
+  },
+  "extra": {
+    "branch-alias": {
+      "dev-develop": "0.1.x-dev"
+    }
+  }
```

Comment:
You could add a [package type](https://getcomposer.org/doc/04-schema.md#type) here